common/include/common/misc/Functional.h

```diff
@@ -42,6 +42,7 @@ class Functional : public Identifiable, public Unique {
     void setFunctions(const j3x::List& func);
     void setDatas(const j3x::List& data);
     void setActivation(Activation activation);
+    static Activation convertActivationStr(const std::string& str);
```

Comment:
move this function to the very bottom (before private: keyword); static functions should be kept separately